index.html

```diff
@@ -117,16 +121,19 @@ <h2 class="visually-hidden">Социальные сети</h2>
         <ul class="socials-list">
           <li class="socials-item">
             <a class="social-link" href="https://twitter.com/htmlacademy_ru">
+              <img src="images/social-twitter.svg" alt="Ссылка на Twitter">
               <span class="visually-hidden">Твиттер</span>
             </a>
           </li>
-          <li class="socials-item">
+          <li class="socials-item">            
             <a class="social-link" href="https://www.facebook.com/htmlacademy">
+              <img src="images/social-facebook.svg" alt="Ссылка на Facebook">
               <span class="visually-hidden">Фейсбук</span>
             </a>
```

Comment:
Тогда не нужен скрытый спан, потому что он повторяют уже существующую информацию. Так же не стоит писать ссылка, потому что скринридер и так скажет что это ссылка, когда она будет в фокусе
```suggestion
            <a class="social-link" href="https://www.facebook.com/htmlacademy">
              <img src="images/social-facebook.svg" alt="Facebook">
            </a>
```